tests/xtro-sharpie/common-AVFoundation.ignore

```diff
@@ -56,35 +56,13 @@
-!extra-null-allowed! 'AVFoundation.AVMutableAudioMixInputParameters AVFoundation.AVMutableAudioMixInputParameters::Create()' has a extraneous [NullAllowed] on return type
-!extra-null-allowed! 'AVFoundation.AVPlayerItem AVFoundation.AVPlayerItem::FromAsset(AVFoundation.AVAsset)' has a extraneous [NullAllowed] on parameter #0
-!extra-null-allowed! 'AVFoundation.AVPlayerItem AVFoundation.AVPlayerItem::FromAsset(AVFoundation.AVAsset,Foundation.NSString[])' has a extraneous [NullAllowed] on parameter #0
-!extra-null-allowed! 'AVFoundation.AVSpeechSynthesisVoice AVFoundation.AVSpeechSynthesisVoice::FromIdentifier(System.String)' has a extraneous [NullAllowed] on parameter #0
-!extra-null-allowed! 'Foundation.NSValue[] AVFoundation.AVPlayerItem::get_SeekableTimeRanges()' has a extraneous [NullAllowed] on return type
 !extra-null-allowed! 'System.Void AVFoundation.AVAudioFormat::.ctor(AudioToolbox.AudioStreamBasicDescription&)' has a extraneous [NullAllowed] on parameter #0
 !extra-null-allowed! 'System.Void AVFoundation.AVAudioFormat::.ctor(AudioToolbox.AudioStreamBasicDescription&,AVFoundation.AVAudioChannelLayout)' has a extraneous [NullAllowed] on parameter #0
 !extra-null-allowed! 'System.Void AVFoundation.AVAudioTime::.ctor(AudioToolbox.AudioTimeStamp&,System.Double)' has a extraneous [NullAllowed] on parameter #0
-!extra-null-allowed! 'System.Void AVFoundation.AVMutableAudioMix::set_InputParameters(AVFoundation.AVAudioMixInputParameters[])' has a extraneous [NullAllowed] on parameter #0
 
 # Initial result from new rule missing-null-allowed
-!missing-null-allowed! 'AVFoundation.AVAsset AVFoundation.AVAssetTrack::get_Asset()' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'AVFoundation.AVAssetTrack AVFoundation.AVPlayerItemTrack::get_AssetTrack()' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'AVFoundation.AVAudioTime AVFoundation.AVAudioTime::ExtrapolateTimeFromAnchor(AVFoundation.AVAudioTime)' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'AVFoundation.AVMetadataItem[] AVFoundation.AVMetadataItem::FilterWithKey(AVFoundation.AVMetadataItem[],Foundation.NSObject,System.String)' is missing an [NullAllowed] on parameter #1
-!missing-null-allowed! 'AVFoundation.AVMetadataItem[] AVFoundation.AVMetadataItem::FilterWithKey(AVFoundation.AVMetadataItem[],Foundation.NSObject,System.String)' is missing an [NullAllowed] on parameter #2
-!missing-null-allowed! 'AVFoundation.AVMutableCompositionTrack AVFoundation.AVMutableComposition::AddMutableTrack(System.String,System.Int32)' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'AVFoundation.AVMutableCompositionTrack AVFoundation.AVMutableComposition::CreateMutableTrack(AVFoundation.AVAssetTrack)' is missing an [NullAllowed] on return type
 !missing-null-allowed! 'System.Boolean AVFoundation.AVPlayerItem::Seek(Foundation.NSDate,AVFoundation.AVCompletion)' is missing an [NullAllowed] on parameter #1
```

Comment:
`AVCompletion` is a `delegate` that is used in several places. Not sure why it is reported as missing `[NullAllowed]` only for this occurrence.

I guess delegates should be nullable by default?